modules/appeals_api/spec/docs/v2/nod_spec.rb

```diff
@@ -0,0 +1,497 @@
+        { apikey: [] }
+      ]
+      consumes 'application/json'
+      produces 'application/json'
+
+      parameter name: :nod_body, in: :body, schema: { '$ref' => '#/components/schemas/nodCreateRoot' }
+
+      parameter in: :body, examples: {
+        'minimum fields used' => {
+          value: JSON.parse(File.read(AppealsApi::Engine.root.join('spec', 'fixtures', 'valid_10182_minimum.json')))
+        },
+        'all fields used' => {
+          value: JSON.parse(File.read(AppealsApi::Engine.root.join('spec', 'fixtures', 'valid_10182.json')))
+        }
+      }
+
+      parameter AppealsApi::SwaggerSharedComponents.header_params[:veteran_ssn_header]
+      let(:'X-VA-SSN') { '000000000' }
+
+      parameter AppealsApi::SwaggerSharedComponents.header_params[:veteran_first_name_header]
+      let(:'X-VA-First-Name') { 'first' }
+
+      parameter AppealsApi::SwaggerSharedComponents.header_params[:veteran_middle_initial_header]
+
+      parameter AppealsApi::SwaggerSharedComponents.header_params[:veteran_last_name_header]
+      let(:'X-VA-Last-Name') { 'last' }
+
+      parameter AppealsApi::SwaggerSharedComponents.header_params[:veteran_birth_date_header]
+      let(:'X-VA-Birth-Date') { '1900-01-01' }
+
+      parameter AppealsApi::SwaggerSharedComponents.header_params[:veteran_file_number_header]
+      parameter AppealsApi::SwaggerSharedComponents.header_params[:consumer_username_header]
+      parameter AppealsApi::SwaggerSharedComponents.header_params[:consumer_id_header]
+
+      response '200', 'Info about a single Notice of Disagreement' do
+        let(:nod_body) do
+          JSON.parse(File.read(AppealsApi::Engine.root.join('spec', 'fixtures', 'valid_10182_minimum.json')))
+        end
+
+        schema AppealsApi::SwaggerSharedComponents.response_schemas[:nod_response_schema]
+
+        before do |example|
+          submit_request(example.metadata)
+        end
+
+        it 'minimum fields used' do |example|
+          assert_response_matches_metadata(example.metadata)
+        end
+
+        after do |example|
+          response_title = example.metadata[:description]
+          example.metadata[:response][:content] = {
+            'application/json' => {
+              examples: {
+                "#{response_title}": {
+                  value: JSON.parse(response.body, symbolize_names: true)
+                }
+              }
+            }
+          }
+        end
+      end
+
+      response '200', 'Info about a single Notice of Disagreement' do
+        let(:nod_body) do
+          JSON.parse(File.read(AppealsApi::Engine.root.join('spec', 'fixtures', 'valid_10182.json')))
+        end
+
+        schema AppealsApi::SwaggerSharedComponents.response_schemas[:nod_response_schema]
+
+        before do |example|
+          submit_request(example.metadata)
+        end
+
+        it 'all fields used' do |example|
+          assert_response_matches_metadata(example.metadata)
+        end
+
+        after do |example|
```

Comment:
Maybe I'm mixing my PRs, but I feel like I've seen this exact `after` block a number of times now. Would be another good candidate for extracting into a helper or two. Perhaps even our own block that can set up the `before`s and `afters` a little more cleanly and just have us get into the real business of `it` tests and not all this boilerplate? Food for thought.